### PR TITLE
Bakursait doxygen

### PR DESCRIPTION
**Pull request by bakursait2**
- Currently addressing issues #70 and #71
- Continuing the work started by others by converting `docstrings` to `doxygen` format.
- Ensuring that each function used an issue is linked to the corresponding issue_number URL in GitHub.
- Formatted the code's maximum line length to comply with [PEP](https://peps.python.org/pep-0008/#maximum-line-length) guidelines, setting it to 79 characters for the code and 72 characters for comments
  - **[Updated]:** As a recommendation from our team member to keep the lines of the code unbroken for better readability, the max line length has been adjusted for code, but still the same for the comments.
- **[New]:** Add class description to each class.

### CODE REVIEW

**Review by Lurk390 — APPROVED** [review]
Look good overall. One comment regarding PEP8 standards, I wouldn't stress too much about the maximum line length of 79. Prioritize readability over this requirement, don't break a line if it will hurt readability. I'm pretty sure the Django and other frameworks recommend a 120 max line length.

File: tempus_fugit_minecraft/sound.py [hunk 1]
```diff
@@ -4,21 +4,46 @@
...
+        """!
+        @brief 
+        Initializes the Sound class.
+        
+        @details If a file path is named, uses that to load the sound 
+        file
+        
+        @param file_path string The file path for the sound file
+        
+        @return None
```
Comment (xieWright) [inline tempus_fugit_minecraft/sound.py:16]: The return should be the initialization of the class.

**Review by xieWright — CHANGES_REQUESTED** [review]
Needs a bit of editing, but overall looks very neatly done. I request the following changes:
1. Remove extraneous text. If there aren't any parameters, we don't need @param. This also applies to @return.
2.  Classes need an @return stating that they return in instance of the class with the specified name.

**Comment by bakursait2** [comment]
> Look good overall. One comment regarding PEP8 standards, I wouldn't stress too much about the maximum line length of 79. Prioritize readability over this requirement, don't break a line if it will hurt readability. I'm pretty sure the Django and other frameworks recommend a 120 max line length.

Good point. I just was trying to follow the Uncle bob's point about how should the length line be. However, You touched readability which is also important. I will reset the code back in the next commit.

Thanks for your suggestion.

**Comment by bakursait2** [comment]
> Look good overall. One comment regarding PEP8 standards, I wouldn't stress too much about the maximum line length of 79. Prioritize readability over this requirement, don't break a line if it will hurt readability. I'm pretty sure the Django and other frameworks recommend a 120 max line length.

The issue fixed.
Thanks

File: tempus_fugit_minecraft/sound.py (hunk 1 above)
Comment (bakursait2) [reply tempus_fugit_minecraft/sound.py]: Both issues fixed. 
Thank you

**Review by xieWright — APPROVED** [review]
Excellent. Looks good. I'll edit the stuff that deals with shaders and sound later.

**Review by criticalglitch — APPROVED** [review]

**Review by w1158mxs — APPROVED** [review]

**Review by criticalglitch — CHANGES_REQUESTED** [review]
Please add links to the respective issues in window.py, then I will approve.

**Comment by bakursait2** [comment]
> Please add links to the respective issues in window.py, then I will approve.

Can you please tell me on what function or what line should I add it? Because I think I added links to all functions that says it resolved by an issue.

**Review by criticalglitch — APPROVED** [review]

**Comment by bakursait2** [comment]
I took a screenshot of the test code result.
I ran the command: `poetry run pytest --cov` and it shows the following:

image

**Pull request merged**